pkg/detectors/appointedd/appointedd.go

```diff
@@ -21,7 +22,7 @@ var (
 	client = common.SaneHttpClient()
 
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives
-	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"appointedd"}) + `\b([a-zA-Z0-9=+]{88})`)
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"appointedd"}) + `\b([a-zA-Z0-9=+]{88})\b`)
```

Comment:
`\b` doesn't work unless the character range is `\w` (`[a-z0-9_`). So if this, or any other pattern, ends in `=` it won't match.

Reply:
There are quite a few regexes like this. I'm wondering if we should use a more strict regex like the gitleaks prefix:
```
(?i)(?:provider)(?:[0-9a-z\-_\t .]{0,20})(?:[\s|']|[\s|"]){0,3}(?:=|>|:{1,3}=|\|\|:|<=|=>|:|\?=)(?:'|\"|\s|=|\x60){0,5}
```

Reply:
I think we'd need more data to determine if that'd be worthwhile. 

Another option would be to do something like this, which IIRC is what GitHub does.
```
[a-zA-Z0-9=+]{88}(?:[^a-zA-Z0-9=+]|\z)
```

Reply:
Assuming appointed is always b64 encoded, which I think it is, the boundary could just be `=` as it's proposed in the current change

https://regex101.com/r/YJeAfP/1